### DISCUSSION TITLE
artwork column (i.e. mp3tag)

### DISCUSSION THREAD

**Discussion by cequalspid** [puddletag/puddletag#883] [Ideas]
When working on a large list of files, it would be helpful if there was a column that let's you quickly see if artwork is missing. Sometimes missing for one track, sometimes an entire album.

This means selecting each album release at a time to see if artwork is missing. Is there something in the columns  field that I am not understanding that I can add?

**Comment by corubba** [comment]
The best I can think of is the `__num_images` field, which gives you the number of images (aka artworks). If there is no image, it is either blank or zero, non-zero otherwise.

**Comment by sandrotosi** [comment]
@cequalspid does that answer your question?

**Reply by cequalspid** [reply]
Oh yes, thank you! That's helpful.
